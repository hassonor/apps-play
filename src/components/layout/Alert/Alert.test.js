import React from "react";
import {mount} from "enzyme";
import Alert from "./Alert";
import {AlertProvider} from "../../../context/alert/AlertContext";

it("expect to render Alert Component", () => {
    const mockUseContext = jest.fn().mockImplementation(() => ({
        alert: {type: 'error', value: 'someError'}
    }));

    React.useContext = mockUseContext;
    expect(mount(<AlertProvider><Alert/></AlertProvider>)).toMatchSnapshot();
});