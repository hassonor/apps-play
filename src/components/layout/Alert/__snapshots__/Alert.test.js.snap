// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Alert Component 1`] = `
<AlertProvider>
  <Alert>
    <p
      className="flex items-start mb-4 space-x-2"
    >
      <svg
        className="w-6 h-6 flex-none mt-0.5"
        fill="none"
        viewBox="0 0 24 24"
      >
        <circle
          cx="12"
          cy="12"
          fill="#FECDD3"
          r="12"
        />
        <path
          d="M8 8l8 8M16 8l-8 8"
          stroke="#B91C1C"
          strokeWidth="2"
        />
      </svg>
      <span
        className="flex-1 text-base font-semibold leading-7 text-white"
      >
        <strong />
      </span>
    </p>
  </Alert>
</AlertProvider>
`;
